styles.css

```diff
@@ -0,0 +1,768 @@
+  background-size: cover;
+  margin: 0 4rem;
+  color: #fff;
+  height: 17rem;
+  width: 30rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.5rem;
+  text-align: center;
+}
+
+.speaker-group {
+  width: 50%;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  margin: 0 1rem 0 1rem;
+}
+
+.speaker-group > div {
+  margin-top: 2.5rem;
+}
+
+.about-img2 > div:hover {
+  display: none;
+}
+
+.about-img1 > div:hover {
+  display: none;
+}
+
+.color-overlay {
+  width: 100%;
+  height: 100%;
+  background: red;
+  opacity: 0.5;
+  position: absolute;
+}
+
+.pro p {
+  font-size: small;
+  width: 80%;
+  height: 100%;
+  max-width: 100%;
+  padding: 0 1rem 0 0.5rem;
+}
+
+.about-img-courses li p {
+  z-index: 1;
+  font-weight: bold;
+}
+
+.para {
+  position: relative;
+  border: 2px solid white;
+  max-width: 45rem;
+  line-height: 1.7;
+  margin-top: 1rem;
+  padding: 2rem;
+  background-color: #c4c4c4;
+  font-weight: 600;
+}
+
+.showcase h2 {
+  position: relative;
+  font-weight: 800;
+  padding-top: 2rem;
+  letter-spacing: 0.2rem;
+}
+
+.para1 {
+  position: relative;
+  max-width: 60%;
+  font-weight: 600;
+  line-height: 1.7;
+  margin-top: 1rem;
+  padding-top: 2rem;
+  color: #909090;
+  font-size: 2rem;
+}
+
+.img-logo {
+  width: 15%;
+  padding-top: 1rem;
+}
+
+hr {
+  background-color: rgb(250, 19, 19);
+  font-weight: bolder;
+  width: 3rem;
+}
+
+.program {
+  padding: 10rem 0 0 0;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  align-items: center;
+  background-color: #272a31;
+  color: #fff;
+}
+
+.courses {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 2rem;
+}
+
+.pro {
+  max-width: 20%;
+  margin: 0.5rem;
+  background-color: #3d4046;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 20rem;
+}
+
+.pro:hover {
+  border: 1px solid #fff;
+}
+
+h5 {
+  font-size: 1rem;
+  font-weight: normal;
+  color: red;
+}
+
+u {
+  padding: 1rem 0 3rem 0;
+}
+
+.footer1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #414246;
+  color: #fff;
+  padding: 1.5rem 0 1.5rem 0;
+  max-width: 100%;
+}
+
+.footer-logos {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer-logos img {
+  margin: 0 0.5rem;
+}
+
+.footer2-img img {
+  width: 4rem;
+}
+
+.footer2 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #d7d7d7;
+  padding: 6rem 0;
+}
+
+.footer2-img {
+  display: flex;
+  align-items: center;
+  margin-left: 4rem;
+}
+
+.footer-desc {
+  margin-right: 4rem;
+}
+
+.footer2-img h6 {
+  font-weight: bolder;
+}
+
+.join-us {
+  display: none;
+}
+
+.speakers {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-top: 7rem;
+  max-width: 100%;
+}
```

Comment:
- Some adjustments also need to be done here in the speakers' section. Let us first take a look at the following screenshots:

| The Provided Design |
| --- |
| ![featured-design](https://user-images.githubusercontent.com/45002654/133526449-ed87ed97-a7f0-4789-a294-29e7507effd1.PNG) |

| The Implementation |
| --- |
| ![featured-imp](https://user-images.githubusercontent.com/45002654/133526366-4872a853-d5b0-4a98-94dc-ded346cbc6a9.PNG) |

## Issues to address
- [ ] In the implementation, this section covers the whole width of the page whilst in the provided design there are margins to the sides of this section. Please make the necessary adjustments.
- [ ] Background applied to the text is not part of the original design. Please on this too.
- [ ] The color of text under the name of every speaker does not match what we have in the original design. These texts are also italicized. Make some adjustments, please.